cfg/cis-1.5/etcd.yaml

```diff
@@ -0,0 +1,131 @@
+---
+controls:
+version: 1.15
+id: 2
+text: "etcd"
+type: "master"
```

Comment:
Is this the right type? Etcd doesn’t necessarily run on the same node as the other control plane (master) components, which is why these tests got split out to a separate section in the CIS benchmark